Dockerfile.builder

```diff
@@ -1,17 +1,17 @@
-FROM node:12.18.4-alpine as builder
+FROM node:17-alpine3.14 as builder
 LABEL maintainer="team@cere.network"
 LABEL description="This is the build stage install all dependecies"
 
 WORKDIR /davinci_nft
 COPY . /davinci_nft
 
 # Install truffle globally and all dependencies
-RUN npm install -g truffle@5.4.7
+RUN npm install -g truffle@latest
```

Comment:
Never use "latest", always versions.

Reply:
I'm removing this branch. Trying to fix CI here